src/Common/tests/TestUtilities/System/PlatformDetection.Unix.cs

```diff
@@ -0,0 +1,246 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.

```

Comment:
I mass-applied the suggestion to update the file headers. I wish VS was smart enough to also do it on Unix. This is the only file that I conditioned for that OS group.